### PR TITLE
remove type:ignore from DataNode.storageType() method #417

### PR DESCRIPTION
**Pull request by ashutuptiwari**
Instead of 
```
return NotImplementedError
```
I did
```
raise NotImplementedError
```
This is for issue #417

### CODE REVIEW

**Comment by ashutuptiwari** [comment]
can i get some help with these build errors @trgiangdo ?

**Comment by trgiangdo** [comment]
Please be patient with me. I'm on it

**Comment by ashutuptiwari** [comment]
Yes of course , thanks for your help

**Review by trgiangdo — APPROVED** [review]
The failing tests are not related to this PR. There is an opening PR that fix those.

I believe this PR is ready to be merged. Thank you a lot for your contribution @ashutuptiwari

**Comment by ashutuptiwari** [comment]
That's a great news! It was fun working on this issue and you @trgiangdo  have been really helpful. 
Also , it would be really great if this PR could be merged before tomorrow so it could count in my hacktoberfest contributions.
Cheers.

**Comment by trgiangdo** [comment]
We will merge it after PR #810 is merged. It should be done by today

**Comment by ashutuptiwari** [comment]
Yay

**Pull request merged**
